apps/etherscan/src/profile.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "etherscan",
-  "displayName": "Etherscan - Contract verification",
-  "description": "Verify Solidity contract code using Etherscan API",
+  "displayName": "Contract verification - Etherscan",
```

Comment:
update for sourcify https://github.com/ethereum/remix-plugins-directory/pull/346